src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -18,48 +17,16 @@ namespace Microsoft.Azure.WebJobs.Script.Binding
     /// </summary>
     internal class WebJobsCoreScriptBindingProvider : ScriptBindingProvider
     {
-        public WebJobsCoreScriptBindingProvider(JobHostConfiguration config, JObject hostMetadata, ILogger logger)
-            : base(config, hostMetadata, logger)
+        public WebJobsCoreScriptBindingProvider(ILogger logger)
+            : base(logger)
         {
         }
 
-        public override void Initialize()
-        {
-            // Apply Blobs configuration
-            var configSection = (JObject)Metadata["blobs"];
-            JToken value = null;
-            if (configSection != null)
-            {
-                if (configSection.TryGetValue("centralizedPoisonQueue", out value))
-                {
-                    Config.Blobs.CentralizedPoisonQueue = (bool)value;
```

Comment:
Where does this configuration happen now for blobs/http?

Reply:
This will be the configuration flow I mentioned today. The various ExtensionOption types will be automatically populated from the loaded configuration.